sass/directives/03_components/_headers.scss

```diff
@@ -254,8 +254,11 @@
   &__headline {
     padding: $header-padding 0;
     max-width: $header-width-normal;
+    position: relative;
```

Comment:
none of the pages have any header images so you should be able to get rid of these extra styles. Going forward we should only have the 5 header types identified under "[Specific Header Types](https://cb-content-enablement.atlassian.net/wiki/spaces/CE/pages/43897344/Header+Audit+5+1+17#HeaderAudit5/1/17-SpecificHeaderTypes)" so if we wanted a header with an image it wouldn't use the small header type